### PR TITLE
Implemented copy locale feature

### LINKED ISSUE #66: Language copy locale overlay
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes

#### Actual Behavior

At the moment this options isn't provided.

#### Expected Behavior

Same as it works in Sulu pages.

### PR DESCRIPTION
**Pull request by trickreich**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #66 
| Related issues/PRs | --
| License | MIT
| Documentation PR | --

#### What's in this PR?

Implemented copy locale feature in list and edit form.

#### Why?

Cause I had to.. :/

### CODE REVIEW

File: Resources/public/js/components/articles/edit/main.js [hunk 1]
```diff
@@ -170,6 +184,9 @@ define([
             this.bindCustomEvents();
             this.showDraftLabel();
             this.setHeaderBar(true);
+            this.loadLocalizations();
+
+            this.options.language = this.options.locale;
```
Comment (wachterjohannes) [inline Resources/public/js/components/articles/edit/main.js]: add a comment why we add this option

File: Resources/public/js/components/articles/edit/main.js [hunk 2]
```diff
@@ -182,7 +199,30 @@ define([
 
             this.sandbox.on('sulu.header.language-changed', function(item) {
                 this.sandbox.sulu.saveUserSetting(this.options.config.settingsKey, item.id);
```
Comment (wachterjohannes) [inline Resources/public/js/components/articles/edit/main.js]: nothing should be done for the same locale

File: Resources/public/js/components/articles/list/main.js [hunk 3]
```diff
@@ -249,6 +276,21 @@ define(['underscore'], function(_) {
...
+        /**
+         * Returns copy article from a given locale to a array of other locales url.
+         *
+         * @param {string} id
+         * @param {string} src
+         * @param {string[]} dest
+         *
+         * @returns {string}
+         */
+        getCopyLocaleUrl: function(id, src, dest) {
```
Comment (wachterjohannes) [inline Resources/public/js/components/articles/list/main.js]: move to service?

File: Resources/public/js/components/articles/edit/main.js [hunk 4]
```diff
@@ -182,7 +199,30 @@ define([
 
             this.sandbox.on('sulu.header.language-changed', function(item) {
```
Comment (wachterjohannes) [inline Resources/public/js/components/articles/edit/main.js]: extract this to own function

**Comment by trickreich** [comment]
@wachterjohannes Have a look once more, thx :)

File: Resources/public/js/components/articles/list/main.js [hunk 5]
```diff
@@ -286,9 +286,7 @@ define([
          * @returns {string}
          */
         getCopyLocaleUrl: function(id, src, dest) {
-            return [
-                '/admin/api/articles/', id, '?locale=', src, '&dest=', dest, '&action=copy-locale'
-            ].join('');
+            return ArticleManager.getCopyLocaleUrl(id,src,dest);
```
Comment (wachterjohannes) [inline Resources/public/js/components/articles/list/main.js]: format this file

File: Resources/public/js/components/articles/edit/main.js [hunk 6]
```diff
@@ -540,9 +559,7 @@ define([
          * @returns {string}
          */
         getCopyLocaleUrl: function(id, src, dest) {
-            return [
-                '/admin/api/articles/', id, '?locale=', src, '&dest=', dest, '&action=copy-locale'
-            ].join('');
+            return ArticleManager.getCopyLocaleUrl(id,src,dest);
```
Comment (wachterjohannes) [inline Resources/public/js/components/articles/edit/main.js]: format this file

**Pull request merged**